requirements/test.txt

```diff
@@ -1,6 +1,4 @@
 bidpom
 pytest-selenium
+pytest-xdist
```

Comment:
You're not pinning pytest at all, which means future releases could break your tests. I think it might be a good idea to pin both pytest and pytest-xdist.